guides/calico.md

```diff
@@ -0,0 +1,80 @@
+# Installing Calico
+
+This should be followed after adding your [Windows node to the cluster with kubeadm](guide-for-adding-windows-node.md#adding-windows-nodes).
+
+## Configuring Calico with Host Process containers
+
+>  **Note:** All code snippets in Linux sections are to be run in a Linux environment on the Linux worker node.
+
+### Prepare Control Plane for Calico
+
+1. Prepare Kubernetes control plane for Calico
+
+Some minor preparation is recommended on the Kubernetes control plane in our cluster. It is recommended to enable bridged IPv4 traffic to iptables chains when using Calico. The following command must be run on all Linux nodes:
+
+```bash
+sudo sysctl net.bridge.bridge-nf-call-iptables=1
+```
+
+2. Download & configure Calico for Linux
+
+The version of Calico will be set in the variable CALICO_VERSION (if another version is needed, modify this variable).
+
+```bash
+export CALICO_VERSION="v3.24.5"
+```
+
+Create the tigera-operator.yaml and the custom-resources.yaml for Calico: 
+```bash
+kubectl create -f https://raw.githubusercontent.com/projectcalico/calico/$CALICO_VERSION/manifests/tigera-operator.yaml
+curl https://raw.githubusercontent.com/projectcalico/calico/$CALICO_VERSION/manifests/custom-resources.yaml -O
+# If IP CIDR differs from 192.168.0.0/16, make sure to modify it in custom-resources.yaml.
+# After the file has the correct CIDR, run this command:
+kubectl create -f custom-resources.yaml
+```
+>  **Note:** Before creating this manifest, read its contents and make sure its settings are correct for your environment. For example, you may need to change the default IP pool CIDR to match your pod network CIDR.
+
+Remove the taints on the master so that you can schedule pods on it.
+
+```bash
+kubectl taint nodes --all node-role.kubernetes.io/control-plane-
+```
+
+It should return the following.
+
+```
+node/<your-hostname> untainted
+```
+
+### Add Windows calico solution
+
+1. Add calico and kube-proxy daemonsets from sig-windows-tools/hostprocess/calico
+
+Now you can add Windows-compatible versions of calico and kube-proxy. In order to ensure that you get a compatible version of kube-proxy, you'll need to substitute the tag of the image. The following example shows usage for Kubernetes v1.25.3, but you should adjust the version for your own deployment.
+
+```bash
+curl -L https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/calico/kube-proxy/kube-proxy.yml | sed 's/KUBE_PROXY_VERSION/v1.25.3/g' | kubectl apply -f -
```

Comment:
nit should we export `KUBE_PROXY_VERSION` and `CALICO_VERSION` env vars at the beginning so it is more obvious which values need to be set by the user?

Reply:
let's follow up with this, I this was a copy/paste of what is there currently to make it easier to follow/maintain